docs/en/relying-party-instance.rst

```diff
@@ -0,0 +1,304 @@
+.. include:: ../common/common_definitions.rst
+
+.. _relying-party-instance:
+
+Relying Party Instance
+=========================
+
+The Relying Party Instance (RPI) is a mobile or embedded application designed to request, receive, and process Digital Credentials from Wallet Instances in a trusted manner. Each RPI ensures the integrity, confidentiality and authenticity of Credential exchanges, enabling secure interactions between Users and Relying Parties.
+
+There are two primary types of Relying Party Instances, each serving different operational environments:
+
+- **Mobile Relying Party Instance**: a native application running on a mobile device (e.g., smartphone or tablet). Each instance corresponds to a specific installation of the application on a device. A Mobile Relying Party Instance functions as a public client, operating on end-user devices where sensitive Credentials cannot be fully protected from potential threats. This implies additional security measures to establish and maintain trust.
```

Comment:
```suggestion
- **Mobile Relying Party Instance**: a native application running on a mobile device (e.g., smartphone or tablet). Each instance corresponds to a specific installation of the application on a device. Since the trust with the Mobile Relying Party Instance is established through the trust established with the Mobile Relying Party Provider, the Mobile Relying Party Instance can be considered a confidential client.
```